canonical-web-and-design/ubuntu.com
# Agressively redirect all getubuntu links to /download

## Done

"getubuntu" links are our biggest source of 404 (~1500/week), with "/getubuntu/download" being the biggest one (~1300 last week).

- This redirects all ubuntu.com/getubuntu.* links to /download

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure /getubuntu redirects to /download
- Ensure /getubuntu/ redirects to /download
- Ensure /getubuntu/download redirects to /download
- Ensure /getubuntu/download/ redirects to /download
- Ensure that /getubuntu/releasenotes redirects to https://wiki.ubuntu.com/Releases
- Ensure that /GetUbuntu/releasenotes still redirects to https://wiki.ubuntu.com/Releases
